### ISSUE TITLE
Azure Redis Cache (azurerm_redis_cache) estimates double cost on Standard SKU

### ISSUE THREAD

**Issue by DavidGSola** [infracost/infracost#2501]
When using a `Standard` SKU on an Azure Redis Cache the cost is double compared to the real one on my production Azure instance and the one provided by Azure Cost Calculator.

In the report generated by Infracost it looks like the `hourlyQuantity` is set to 2. I have not found any configuration on the terraform resource `azurerm_redis_cache` to modify the number of nodes on the cache. Anyway according to the MSFT documentation the cost of the `Standard` SKU is for two-nodes (see https://azure.microsoft.com/en-us/pricing/details/cache/).

### How to reproduce the issue:

Terraform resource:
```tf
resource "azurerm_redis_cache" "redis" {
  capacity            = 0
  family              = "C"
  location            = "northeurope"
  name                = "redis"
  resource_group_name = "resource_group"
  sku_name            = "Standard"
}
```

`azurerm_redis_cache` with `Basic` SKU are correctly estimated.

### Expected result:

Infracost estimates the montly cost of a `Standard` Azure Redis Cache with a closer value to the real one.

### Actual result:

Infracost is estimating the monthly cost two times the actual one. `hourlyQuantity` is set to 2 and `monthlyQuantity` to 1460.

```json
{
   "name": "azurerm_redis_cache.redis",
    "metadata": {
        "calls": [
            {
                "blockName": "azurerm_redis_cache.redis",
                "filename": "main2.tf"
            }
        ],
        "filename": "main2.tf"
    },
    "hourlyCost": "0.1017797",
    "monthlyCost": "74.299181",
    "costComponents": [
        {
            "name": "Cache usage (Standard_C0)",
            "unit": "hours",
            "hourlyQuantity": "2",
            "monthlyQuantity": "1460",
            "price": "0.05088985",
            "hourlyCost": "0.1017797",
            "monthlyCost": "74.299181"
        }
    ]
}
```

**Comment by aliscott** [comment]
Thanks for reporting @DavidGSola 🙏. I've added this to our next sprint.

**Comment by aliscott** [comment]
Thanks for reporting this @DavidGSola 🙏 , this is now released in [v0.10.24](https://github.com/infracost/infracost/releases/tag/v0.10.24).
